.travis.yml

```diff
@@ -130,6 +130,7 @@ matrix:
         - PG_VERSION=9.2
         - ZULU_JDK=7
         - MCENTRAL=Y
+        - COVERAGE=Y
```

Comment:
There are 10 `COVERAGE=Y` lines and `after_n_builds` is 8.
How does that play together?

PS. Could you add a test that compares `after_n_builds` vs number of COVERAGE jobs?

Reply:
mmm... I must forgot update after_n_builds

> PS. Could you add a test that compares after_n_builds vs number of COVERAGE jobs?
Something like another job?

Reply:
Something like a line in https://github.com/pgjdbc/pgjdbc/blob/master/.travis/travis_build.sh#L78-L82

Reply:
I actually added a test to break the build before anything, this will work like a fail-fast test since is useless to test to the end and break there.